code/nrf-connect/samples/zigbee/src/restart_handler.c

```diff
@@ -0,0 +1,14 @@
+#include "restart_handler.h"
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zigbee/zigbee_app_utils.h>
+
+LOG_MODULE_REGISTER(restart_handler, CONFIG_LOG_DEFAULT_LEVEL);
+
+static void restart_network_steering_cb(struct k_timer *timer) {
+  LOG_DBG("Restart handler expired. Restarting network steering.");
+  user_input_indicate();
```

Comment:
Interesting name Nordic chose for [this function](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/zigbee/zigbee_app_utils.html#c.user_input_indicate). I'd add a little comment:

`// If the device is not commissioned, the rejoin procedure is started.`